modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java

```diff
@@ -1132,6 +1322,87 @@ private Folder _getFolder(NotificationQueueEntry notificationQueueEntry)
 				notificationQueueEntry.getNotificationQueueEntryId()));
 	}
 
+	private Set<String> _getTermNames(
+			long objectDefinitionId, Set<String> selectedTermCategories)
+		throws Exception {
+
+		Set<String> termNames = new HashSet<>();
+
+		MockLiferayResourceRequest mockLiferayResourceRequest =
+			new MockLiferayResourceRequest();
+
+		mockLiferayResourceRequest.addParameter(
+			"objectDefinitionId", String.valueOf(objectDefinitionId));
+
+		MockLiferayResourceResponse mockLiferayResourceResponse =
+			new MockLiferayResourceResponse();
+
+		_mvcResourceCommand.serveResource(
+			mockLiferayResourceRequest, mockLiferayResourceResponse);
+
+		ByteArrayOutputStream byteArrayOutputStream =
+			(ByteArrayOutputStream)
+				mockLiferayResourceResponse.getPortletOutputStream();
+
+		JSONArray jsonArray = JSONFactoryUtil.createJSONArray(
+			byteArrayOutputStream.toString());
```

Comment:
@devpedrotavares rename this to `ftlTermCategoriesJSONArray`, besides the resource command is `/notification_templates/notification_template_ftl_elements` I would not rename this to `ftlElementsJSONArray` because `ftl term categories` is more clear in my opinion, maybe we should rename this endpoint to `/notification_templates/notification_template_ftl_term_categories` but let's do this in a follow up pull request, this one is too big.

Reply:
Okay!